src/Mod/Path/Path/Op/Helix.py

```diff
@@ -138,7 +138,7 @@ def initCircularHoleOperation(self, obj):
             "Helix Drill",
             QT_TRANSLATE_NOOP(
                 "App::Property",
-                "Extra value to stay away from final profile- good for roughing toolpath",
+                "Extra value to stay away from final profile- good for roughing toolpath"
```

Comment:
These comma cleanups are a bit suspicious. I presume you manually removed them? The formatting has been given to `black` so you might want to run that over this file.

Reply:
Probably I have removed it. I  will put it back.

Trailing commas in python argument lists are optional.
In FreeCAD most uses of QT_TRANSLATE_NOOP do not use a trailing comma. Few do.

Reply:
Added commit 892933c971f33ec682fa6b7a3c22dab5216453c2 "Undo removal of comma in argument list of QT_TRANSLATE_NOOP."

Reply:
@mlampert anything else?

Reply:
@luzpaz I gave my approval a while ago.